engine/engine_api_test.go

```diff
@@ -30,6 +31,21 @@ func (e *Engine) LinkSystem() *ipld.LinkSystem {
 	return &e.lsys
 }
 
+// Returns the engine's default provider ID, exposed for testing purposes only.
```

Comment:
Side note: godoc convention is to start the doc with the name of the function.

Reply:
Fixed